atom/browser/native_window_views_win.cc

```diff
@@ -299,6 +300,59 @@ bool NativeWindowViews::PreHandleMSG(UINT message,
   }
 }
 
+void NativeWindowViews::HandleSizingEvent(WPARAM w_param, LPARAM l_param) {
+  double aspect_ratio = GetAspectRatio();
+  if (fabs(aspect_ratio) < 0.0001) {
+    return;
+  }
+
+  RECT* window_rect;
+  window_rect = reinterpret_cast<RECT*>(l_param);
```

Comment:
nit: assign and declare together
```suggestion
  RECT* window_rect = reinterpret_cast<RECT*>(l_param);
```